resources/create-database-script.sql

```diff
@@ -16,9 +16,45 @@ create table `User` (
     FOREIGN KEY (roleID) REFERENCES Role(roleID)
 );
 
-create table Token (
-	userID SMALLINT NOT NULL,
-    token varchar(64) NOT NULL,
-    expiry DATETIME NOT NULL,
-    FOREIGN KEY (userID) REFERENCES User(userID)
-);
\ No newline at end of file
+CREATE TABLE `Capability` (
+    capabilityID SMALLINT NOT NULL AUTO_INCREMENT,
+    capabilityName VARCHAR(64) NOT NULL,
+    PRIMARY KEY (capabilityID)
+);
+
+insert into Capability(capabilityID, capabilityName) VALUES (1, 'Engineering');
+insert into Capability(capabilityID, capabilityName) VALUES (2, 'Cyber  Security');
+insert into Capability(capabilityID, capabilityName) VALUES (3, 'Data & AI');
+
+
+CREATE TABLE `Bands` (
+    bandID SMALLINT NOT NULL AUTO_INCREMENT,
+    bandName VARCHAR(64) NOT NULL,
+    PRIMARY KEY (bandID)
+);
+
+insert into Bands(bandID, bandName) VALUES (1, 'Apprentice');
+insert into Bands(bandID, bandName) VALUES (2, 'Trainee');
+insert into Bands(bandID, bandName) VALUES (3, 'Associate');
+insert into Bands(bandID, bandName) VALUES (4, 'Senior Associate');
```

Comment:
Something that may help later. 

Kainos structures its bands with 1 being the top (principle) and band 7 being the bottom (apprentice). It may help you remember that if the bands match the same way as Kainos does them (So insert all of them with 1 being Principle)